app/views/sessions/new.html.erb

```diff
@@ -11,6 +11,11 @@
       <%= f.label :password %>
       <%= f.password_field :password, class: 'form-control' %>
 
+      <%= f.label :remember_me, class: "checkbox inline" do %>
```

Comment:
It depends. While my original comment was not to force a change, it was more to point out that you may want to consider simplify the need to have the customer "check" the "Remember Me" button if the situation presents itself. The classic example is Amazon.com. If you sign-on, your session exists for quite a long time. Unless you actually need to manage your account, check your orders, etc. you don't have to log in again. An example of the convenience is that if prime customers want to go straight to watching videos, they don't have to log in every 30 or so minutes.

In general, while the resource you're going off of may mention the "Remember Me," you may want to consider if this can be simplified. For example, if you are making an in-office application, would that be used in a public computer? If not, then, is it still worthwhile? It might be or it might not, based on the privileges you allow. Overall, it's a design choice you may want to make based off your requirements and what you deem to be secure. Again, the point of the comment was more to push for considering the consequences of this, which in a real-world system you would decide during the design phase, rather than asking you to make a change.